Get_Time_TimezoneWise/requirements.txt

```diff
@@ -0,0 +1,159 @@
+absl-py==0.13.0
```

Comment:
Why there are so many libraries in `requirements.txt` ?

Reply:
Updated requirements.txt